include/aperturedb/VDMSClient.h

```diff
@@ -83,6 +113,8 @@ namespace VDMS {
 
     class VDMSClient {
     public:
+        // Deprecated multi-parameter ctors.
+        // Prefer VDMSClientConfig ctor.
```

Comment:
Yeah, I like that.  My main objection to the multi-param ctors was that they used a bunch of positional parameters of generic types, which can lead to errors like `VDMSClient client(username, password, 55555);` which compiles but interprets the first two args incorrectly as `api_key` and `addr`.

I like your suggestion because it addresses both concerns. 👍